src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/http/AuthorizationServerClient.kt

```diff
@@ -178,11 +186,12 @@ internal class AuthorizationServerClient(
                 state(State(state))
                 issuerState?.let { customParameter("issuer_state", issuerState) }
                 if (scopes.isNotEmpty()) {
-                    scope(NimbusScope(*scopes.map { it.value }.toTypedArray() + "openid"))
+                    scope(NimbusScope(*scopes.map { it.value }.toTypedArray()))
                 }
                 if (credentialsConfigurationIds.isNotEmpty()) {
                     authorizationDetails(credentialsConfigurationIds.map(::toNimbus))
                 }
+                prompt(Prompt.Type.LOGIN)
```

Comment:
Something minor: Given that the Authorization Server might not advertise PAR support, and `ParUage` might be set to `Always`, we could add a check that `parEndpoint` is indeed not null.

```kotlin
val parEndpoint = requireNotNull(authorizationServerMetadata.pushedAuthorizationRequestEndpointURI)
```

Even without this, at the end of the day the constructor call to `PushedAuthorizationRequest` will fail non the less.

Reply:
So, perhaps this means that "Always" doesn't make sense.

Reply:
@dzarras Please check